app/views/organizations/index.html.erb

```diff
@@ -15,12 +15,52 @@
     </div>
   <% end %>
 
-  <h2>Default stream statistics by provider</h2>
+  <!-- Only show consumer-only table to admins -->
+  <% if current_user && current_user.has_role?(:admin) %>
```

Comment:
Maybe this should only show up if there are `@consumers`?

Reply:
@corylown added a commit to address this.